### DISCUSSION TITLE
How to extend the theme to implement color Schemes for Modal

### DISCUSSION THREAD

**Discussion by JakovGlavac** [chakra-ui/chakra-ui#4590] [Q&A] [answered]
Hi, I'm trying to change modal color scheme, but when I use property 		
```
<Modal
	colorScheme="red" // ⬅ it looks like this
	initialFocusRef={initialRef}
	finalFocusRef={finalRef}
	isOpen={isOpen}
	onClose={onClose}
>
```
nothing changes
![image](https://user-images.githubusercontent.com/34971091/129890223-79b56036-7bbd-43d9-bec0-5b8750cf6b05.png)

According to the [docs](https://chakra-ui.com/docs/overlay/modal)

> Color Schemes for Modal are not implemented in the default theme. You can extend the theme to implement them

after looking at docs on how to "[Customize Theme](https://chakra-ui.com/docs/theming/customize-theme#theme-extension-withdefaultcolorscheme)" but I cant still figure out how to do it.

**Comment by aaronkik** [comment, marked as answer]
Hi,

I've created a sandbox link on how I acheived it: https://codesandbox.io/s/extend-colorscheme-gquof?file=/modal.js

If you look at the components on the chakra ui website that use a `colorScheme` then click the view source button at the top of the page you can get an idea of how the team have constructed it like the [Button](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/button.ts) component
